Lib9c/Model/Mail/OrderBuyerMail.cs

```diff
@@ -0,0 +1,39 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Bencodex.Types;
+using Nekoyume.Model.State;
+using static Lib9c.SerializeKeys;
+
+namespace Nekoyume.Model.Mail
+{
+    [Serializable]
+    public class OrderBuyerMail : Mail
+    {
+        public readonly Guid OrderId;
```

Comment:
`public override MailType MailType => MailType.Auction;`
Please override the mail type